doc/getting_started/installation_linux.rst

```diff
@@ -70,6 +70,10 @@ Install additional packages required for development with Zephyr::
    cd ~/zephyr  # or to your directory where zephyr is cloned
    pip3 install --user -r scripts/requirements.txt
 
+To install pyocd, which is a python2 module, please use::
```

Comment:
how about:

    If you require pyocd, (an open source python2 library for programming and
    debugging ARM Cortex-M microcontrollers), use this command::